src/presence_analyzer/templates/base.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html lang=en>
+<head>
+    <meta charset=utf-8>
+    {% block title %}{% endblock %}
+    <meta name="keywords" content="" />
+    <meta name="description" content=""/>
+    <meta name="author" content="STX Next sp. z o.o."/>
+    <meta name="viewport" content="width=device-width; initial-scale=1.0">
+    <link href="{{ url_for('static', filename='css/normalize.css') }}" media="all" rel="stylesheet" type="text/css" />
+    <link href="{{ url_for('static', filename='css/main.css') }}" media="all" rel="stylesheet" type="text/css" />
+
+     <script src="{{ url_for('static', filename='js/jquery.min.js') }}"></script>
+    <script type="text/javascript" src="https://www.google.com/jsapi"></script>
+    {% block head %}{% endblock %}
+</head>
+
+<body>
+<div id="main">
+        <div id="header">
+            <h1>Presence analyzer</h1>
+            <ul>
+            {%- for name, link in pages.items() -%}
+                {%- if link == '/'+request.url.rsplit('/', 1)[1] -%}
+                <li id="selected"><a href="{{link}}">{{name}}</a></li>
+                {%- else -%}
+                <li><a href="{{link}}">{{name}}</a></li>
+                {%- endif -%}
+            {%- endfor -%}
+            </ul>
+        </div>
+        <div id="content">
+            {%- for name, link in pages.items() -%}
+                {%- if link == '/'+request.url.rsplit('/', 1)[1] -%}
+                <h2>{{name}}</h2>
```

Comment:
`{{ name }}`
Indent

Wont be easier to send current pair in context to the template and just apply it here and in generating menu items and adding active class? You're repeating this loop.